.env.example

```diff
@@ -0,0 +1,6 @@
+PORT=NUMBER
```

Comment:
better to use some default values here (ex: PORT=3000, etc.)

Reply:
fixed it